TestKeystoreDir In lib/utils/utils_test.go Fails On Windows Machine

Paths aren't normalized across OS and thus the test fails on Windows OS. Can fix by cleaning the constructed path.

```
=== RUN   TestKeystoreDir
    utils_test.go:61: 
        	Error Trace:	gossamer/lib/utils/utils_test.go:61
        	Error:      	Not equal: 
        	            	expected: "C:\\Users\\User\\AppData\\Local\\Temp\\TestKeystoreDir1969112655\\001/keystore"
        	            	actual  : "C:\\Users\\User\\AppData\\Local\\Temp\\TestKeystoreDir1969112655\\001\\keystore"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-C:\Users\User\AppData\Local\Temp\TestKeystoreDir1969112655\001/keystore
        	            	+C:\Users\User\AppData\Local\Temp\TestKeystoreDir1969112655\001\keystore
        	Test:       	TestKeystoreDir
--- FAIL: TestKeystoreDir (0.00s)
```

```go
func TestKeystoreDir(t *testing.T) {
	testDir := t.TempDir()

	keystoreDir, err := KeystoreDir(testDir)
	require.NoError(t, err)

	assert.Equal(t, testDir+"/keystore", keystoreDir)
}
```

Solution:

```go
	testDir := t.TempDir()

	keystoreDir, err := KeystoreDir(testDir)
	require.NoError(t, err)

	assert.Equal(t, filepath.Clean(testDir+"/keystore"), keystoreDir)
```

--- a/lib/utils/utils_test.go
+++ b/lib/utils/utils_test.go
@@ -4,6 +4,7 @@
 package utils
 
 import (
+	"path/filepath"
 	"strings"
 	"testing"
 
@@ -58,5 +59,5 @@ func TestKeystoreDir(t *testing.T) {
 	keystoreDir, err := KeystoreDir(testDir)
 	require.NoError(t, err)
 
-	assert.Equal(t, testDir+"/keystore", keystoreDir)
+	assert.Equal(t, filepath.Join(testDir, "keystore"), keystoreDir)
 }

